articles/quickstart/native/ios-swift/interactive.md

```diff
@@ -125,7 +158,7 @@ For further reference on Carthage, check their [official documentation](https://
 
 The Auth0.swift SDK needs your Auth0 **domain** and **Client ID**. You can find these values in the [settings page](${manage_url}/#/applications/${account.clientId}/settings) of your Auth0 application.
 
-- **domain**: The domain of your Auth0 tenant. If you are using a [custom domain](/customize/custom-domains), you should set this to the value of your custom domain instead.
+- **domain**: The domain of your Auth0 tenant. If you have a [custom domain](/customize/custom-domains), use it instead of your Auth0 tenant’s domain.
```

Comment:
Nit: I think using `this` vs `it` here has greater emphasis on using the custom domain.

```suggestion
- **domain**: The domain of your Auth0 tenant. If you have a [custom domain](/customize/custom-domains), use this instead of your Auth0 tenant’s domain.
```